webapp/src/components/sidebar_header/sidebar_header.jsx

```diff
@@ -6,11 +6,12 @@ import SidebarButtons from '../sidebar_buttons';
 export default class SidebarHeader extends React.PureComponent {
     static propTypes = {
         show: PropTypes.bool.isRequired,
+        connected: PropTypes.bool.isRequired,
         theme: PropTypes.object.isRequired,
     };
 
     render() {
-        if (!this.props.show) {
+        if (!this.props.show || !this.props.connected) {
```

Comment:
Does this also need to be applied to the team sidebar version of this component (which is shown when the user is member of multiple teams)? Also, do you know if this was always this way, or was this a regression?

Reply:
The team sidebar version also uses the same code, so it is applied to that as well, and we just see the `GitLab` icon to connect our account.
Also, this was always this way, we have fixed the issue mentioned in the description.